### ISSUE TITLE
[r] Failed to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#4475]
```
checking whether clang -g -O2 can compile C++ code... no
checking how to run the C++ preprocessor... /lib/cpp
configure: error: in `/usr/local/tmp/crew/R-4.0.3.tar.gz.dir/R-4.0.3':
configure: error: C++ preprocessor "/lib/cpp" fails sanity check
See `config.log' for more details
r failed to build: `./configure --prefix=/usr/local --libdir=/usr/local/lib --disable-maintainer-mode --enable-R-shlib --with-x` exited with 1
```

**Comment by satmandu** [comment]
Why is clang being used?

Looks like your llvm installation is borked?

**Comment by satmandu** [comment]
(also I had to reinstall libiconv before I could compile r due to iconv.h not being the libiconv version.)

**Comment by satmandu** [comment]
Just got r compiled on arm7l.

I used gcc10, and also this:
```
  if ARCH =~ /^(arm7l|aarch64)$/
    ENV['CC'] = 'gcc'
    ENV['CXX'] = 'g++'
    ENV['CFLAGS'] = '-march=armv7-a'
    ENV['CXXFLAGS'] = '-march=armv7-a'
  end
```

**Comment by uberhacker** [comment]
If you need to recompile packages, those should be included in the pull request also.  I have no idea why clang is used.

**Comment by uberhacker** [comment]
Fixed in #4485.
